### PR TITLE
Support for clearing rows

### PR DESCRIPTION
**Pull request by ashmckenzie**
* Added `Table.ClearRows()`
* Added testing for golang 1.5 -> 1.8
* Tidied up `table_test.go`

### CODE REVIEW

**Comment by ashmckenzie** [comment]
@olekukonko @mattn bump :)

File: table_test.go [hunk 1]
```diff
@@ -462,3 +505,59 @@ func TestAutoMergeRows(t *testing.T) {
...
+
+	buf.Reset()
+	table.ClearRows()
+	table.Render()
+
+	want = `+----------+-------------+-------+---------+
+|   DATE   | DESCRIPTION |  CV2  | AMOUNT  |
++----------+-------------+-------+---------+
++----------+-------------+-------+---------+
+|                          TOTAL | $145 93 |
```
Comment (mattn) [inline table_test.go]: Thanks for your contribution. It seems pretty good. I'm not sure but why this summary value is remaining?

Comment (ashmckenzie) [inline table_test.go]: Ah, you're exactly correct, I've missed something.  Thank-you, patching incoming :)

**Comment by mattn** [comment]
LGTM, I leave for others.

**Comment by mattn** [comment]
Hmm, I say LGTM again.

**Comment by olekukonko** [comment]
@mattn ... Well done :)

**Pull request merged**